src/components/HeaderView.jsx

```diff
@@ -62,26 +62,26 @@ const HeaderView = ({ schedulerData, nonAgendaCellHeaderTemplateResolver }) => {
         : { width: cellWidth };
       if (index === headers.length - 1) style = !!item.nonWorkingTime ? { color: config.nonWorkingTimeHeadColor, backgroundColor: config.nonWorkingTimeHeadBgColor } : {};
       const cellFormat = cellUnit === CellUnit.Week
-          ? config.nonAgendaWeekCellHeaderFormat
-          : cellUnit === CellUnit.Month
-            ? config.nonAgendaMonthCellHeaderFormat
-            : cellUnit === CellUnit.Year
-              ? config.nonAgendaYearCellHeaderFormat
-              : config.nonAgendaOtherCellHeaderFormat;
-      const pFormattedList = cellFormat.split('|').map(dateFormatPart => datetime.format(dateFormatPart));
+        ? config.nonAgendaWeekCellHeaderFormat
+        : cellUnit === CellUnit.Month
+        ? config.nonAgendaMonthCellHeaderFormat
+        : cellUnit === CellUnit.Year
```

Comment:
Here I added spaces intentionally. When ternary operators are nested I think that is a better way to determine the logic.